src/app/loaders/express/helmet.ts

```diff
@@ -89,9 +87,6 @@ const helmetMiddlewares = () => {
   // https://github.com/helmetjs/helmet/blob/cb170160e7c1ccac314cc19d3b979cfc771f1349/middlewares/content-security-policy/index.ts#L135
   if (reportUri) cspOptionalDirectives.reportUri = [reportUri]
 
-  // Add on upgradeInsecureRequest CSP header if !config.isDev
-  if (!config.isDev) cspOptionalDirectives.upgradeInsecureRequests = []
```

Comment:
do we now need to do some handling for dev environment since the conditionals are flipped?

Reply:
thanks! added in